Sming/system/include/debug_progmem.h

```diff
@@ -44,6 +44,11 @@ extern "C" {
 #define DEBUG_VERBOSE_LEVEL INFO
 #endif
 
+// Dummy to omit debug information safely
+#define debug_none(fmt, ...)                                                                                           \
```

Comment:
It would be great if the compiler complete removes debug code when in SMING_RELEASE mode. That used to be the case with the previous code. Can you check if that is the case also with this PR?

Reply:
Built complex application with `SMING_RELEASE=1`, with debug_none:

```
   Section|                   Description| Start (hex)|   End (hex)|Used space
------------------------------------------------------------------------------
      data|        Initialized Data (RAM)|    3FFE8000|    3FFE8388|     904
    rodata|           ReadOnly Data (RAM)|    3FFE8390|    3FFE906C|    3292
       bss|      Uninitialized Data (RAM)|    3FFE9070|    3FFEF910|   26784
      text|            Cached Code (IRAM)|    40100000|    40106FDA|   28634
irom0_text|           Uncached Code (SPI)|    40202010|    4026B826|  432150
Total Used RAM : 30980
Free RAM : 50940
Free IRam : 4152
```

without debug_none:

```
   Section|                   Description| Start (hex)|   End (hex)|Used space
------------------------------------------------------------------------------
      data|        Initialized Data (RAM)|    3FFE8000|    3FFE8388|     904
    rodata|           ReadOnly Data (RAM)|    3FFE8390|    3FFE906C|    3292
       bss|      Uninitialized Data (RAM)|    3FFE9070|    3FFEF910|   26784
      text|            Cached Code (IRAM)|    40100000|    40106FDA|   28634
irom0_text|           Uncached Code (SPI)|    40202010|    4026B826|  432150
Total Used RAM : 30980
Free RAM : 50940
Free IRam : 4152
```

Reply:
Ok, so it seems identical. Which is good. Can you check also what are the stats with DEBUG_VERBOSE_LEVEL=3?
```
cd %SMING_HOME%
set DEBUG_VERBOSE_LEVEL=3
make dist-clean
# go to your app if needed and then 
make 
```

Reply:
```
   Section|                   Description| Start (hex)|   End (hex)|Used space
------------------------------------------------------------------------------
      data|        Initialized Data (RAM)|    3FFE8000|    3FFE8388|     904
    rodata|           ReadOnly Data (RAM)|    3FFE8390|    3FFEA06C|    7388
       bss|      Uninitialized Data (RAM)|    3FFEA070|    3FFF0970|   26880
      text|            Cached Code (IRAM)|    40100000|    40106F32|   28466
irom0_text|           Uncached Code (SPI)|    40202010|    4027479E|  468878
Total Used RAM : 35172
Free RAM : 46748
Free IRam : 4320
```